.github/scripts/generate_binary_build_matrix.py

```diff
@@ -360,12 +374,13 @@ def generate_wheels_matrix(
                             ".", "_"
                         ),
                         "pytorch_extra_install_requirements":
-                        PYTORCH_EXTRA_INSTALL_REQUIREMENTS["12.1"]  # fmt: skip
+                        PYTORCH_EXTRA_INSTALL_REQUIREMENTS["12.4"]  # fmt: skip
```

Comment:
```suggestion
                        PYTORCH_EXTRA_INSTALL_REQUIREMENTS["12.1"]  # fmt: skip
```
We need to keep 12.1 here still since this is the wheel we will be uploading to pypi 12.4 should still be experimental build for now.